cohorts/functions.py

```diff
@@ -14,40 +14,58 @@
 
 from __future__ import print_function
 
+from .variant_filters import variant_qc_filter, effect_qc_filter, neoantigen_qc_filter
+
 import numpy as np
 from varcode.effects import Substitution
 
-def snv_count(row, cohort, **kwargs):
+def snv_count(row, cohort, qc_filter=False, **kwargs):
```

Comment:
Seemed easier to say `qc_filter=True` than worry about which function to provide?

Separately, I think it probably makes sense to default to `True`, but that broke tests so I punted on it for now.